src/Search/IndexPage.jsx

```diff
@@ -3,7 +3,7 @@ const APPLICATION_ID = props.appId ?? "B6PI9UKKJT";
 const INDEX = props.index ?? "replica_prod_near-social-feed";
 const API_URL = props.apiUrl ?? `https://${APPLICATION_ID}-dsn.algolia.net/1/indexes/${INDEX}/query?`;
 const INITIAL_PAGE = props.initialPage ?? 0;
-const facets = props.facets ?? ["All", "People", "Apps", "Components", "Posts"];
+const facets = props.facets ?? ["All", "Apps", "Components"];
```

Comment:
what is our stand on Components? Because we said in the `docs` that we are deprecating them

Do we want people to find random components from random users when we advise against it security on docs and the upcoming blogpost?